src/graph_config_dialog.js

```diff
@@ -273,16 +285,18 @@ export function GraphConfigurationDialog(dialog, onSave) {
         $('select.graph-height', graphElem).replaceWith(chooseHeight(graph.height?(graph.height):1));
 
         // Add Field List
-        for (var i = 0; i < graph.fields.length; i++) {
-            var
-                field = graph.fields[i],
-                fieldElem = renderField(flightLog, field, field.color?(field.color):(GraphConfig.PALETTE[i].color));
-
-            fieldList.append(fieldElem);
+        for (let i=0; i<graph.fields.length; ++i) {
```

Comment:
- Did you intent to start with ++i ?
- Operators needs space for better readability

```suggestion
        for (let i = 0;  i < graph.fields.length; i++) {
```